### DISCUSSION TITLE
"All Countries" option

### DISCUSSION THREAD

**Discussion by kuyucuburak** [Daniel-Ioannou/flutter_country_picker#57] [Ideas]
Hi,

Firstly, thank you for this great library man!

Secondly, for a feature in my app, users must be able to select "all countries" option. Am I able to add my own optio like this to the country list? If not, I will fork and add it. After I implemented that, should I create a branch in this library? That could be useful for others.

Best

**Comment by Daniel-Ioannou** [comment]
This feature was released at [v2.0.11](https://pub.dev/packages/country_picker/versions/2.0.11). 
Thank @kuyucuburak for your contribution

**Comment by kuyucuburak** [comment]
Your welcome :)
